NebulaModel/Attributes/UIRangeAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace NebulaModel.Attributes
+{
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public class UIRangeAttribute : Attribute
+    {
+        public readonly float min;
+        public readonly float max;
+
+        public UIRangeAttribute(float min, float max)
```

Comment:
Can use: https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.rangeattribute?view=netframework-3.5

Reply:
Oh wow I didn't knew those existed. I'll change the code to use them

Reply:
humm, I'll keep my UIRangeAttribute, otherwise we need to include the entire DataAnnotation dll with the mod. Too overkill for our usage.